src/apps/vpn/models/subscriptiondata.cpp

```diff
@@ -222,6 +224,16 @@ void SubscriptionData::writeSettings() {
   SettingsHolder::instance()->setSubscriptionData(m_rawJson);
 }
 
+bool SubscriptionData::plusTax() {
+  MozillaVPN* vpn = MozillaVPN::instance();
+  QString countryCode = vpn->location()->countryCode();
```

Comment:
This is still wrong. The location should not be used here. As an European user i don't want to see +tax if i fly to Canada, right?

Reply:
This is only used before a subscription is purchased (on the IAP screen). What are you suggesting we use instead?